### PR TITLE
ALP-97: Sign in Active Directory

### PR DESCRIPTION
**Pull request by thaingocnguyen**

### CODE REVIEW

File: aunt-leahs-app/src/pages/AdminShiftDataPage.jsx [hunk 1]
```diff
@@ -82,4 +86,8 @@ const AdminShiftDataPage = () => {
 	);
 };
 
-export default AdminShiftDataPage;
\ No newline at end of file
+export default withAuthentication(AdminShiftDataPage, {
+	provider: authProvider,
+	reduxStore: store,
+	forceLogin: true
+});
```
Comment (justinkkwan) [inline aunt-leahs-app/src/pages/AdminShiftDataPage.jsx]: Add new line

File: aunt-leahs-app/src/redux/auth/authReducer.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      return {
+        ...state,
+        state: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+export default authReducer;
```
Comment (justinkkwan) [inline aunt-leahs-app/src/redux/auth/authReducer.js]: New line

File: aunt-leahs-app/src/redux/rootReducer.js [hunk 3]
```diff
@@ -3,9 +3,25 @@ import { combineReducers } from 'redux';
...
+    }
+
+    return appReducer(state, action);
+}
+
+export const clearStateAction = () => (
+    {
+        type: 'CLEAR_STATE'
+    }
+)
```
Comment (justinkkwan) [inline aunt-leahs-app/src/redux/rootReducer.js]: new line

**Review by justinkkwan — APPROVED** [review]
Some new line stuff but yeah 👍

**Review by Leonlui — APPROVED** [review]
👍👍

**Pull request merged**